pkg/otelcollector/metricsprocessor/internal/sdk_labels.go

```diff
@@ -13,8 +14,19 @@ import (
 func AddSDKSpecificLabels(attributes pcommon.Map) {
 	// Compute durations
 	flowStart, flowStartExists := getSDKLabelTimestampValue(attributes, otelconsts.ApertureFlowStartTimestampLabel)
+	if !flowStartExists {
+		flowStart, flowStartExists = getSDKLabelTimestampValue(attributes, otelconsts.ApertureFlowStartTimestampLabelMs)
```

Comment:
Should prefer ms timestamp over the existing one. Can deprecate the older one after few versions

Reply:
Sure, will make the change

Reply:
@tanveergill Handled in [0cb665f](https://github.com/fluxninja/aperture/pull/2604/commits/0cb665f3fddb0b936690cc6bc151cbd3a22aea54)